templates/instance/emailblocklist.rs.html

```diff
@@ -0,0 +1,66 @@
+@use templates::base;
+@use plume_models::blocklisted_emails::BlocklistedEmail;
+@use template_utils::*;
+@use routes::*;
+
+@(ctx:BaseContext, emails: Vec<BlocklistedEmail>, page:i32, n_pages:i32)
+    @:base(ctx, i18n!(ctx.1, "Blocklisted Emails"), {}, {}, {
+    @tabs(&[
+        (&uri!(instance::admin).to_string(), i18n!(ctx.1, "Configuration"), false),
+        (&uri!(instance::admin_instances: page = _).to_string(), i18n!(ctx.1, "Instances"), false),
+        (&uri!(instance::admin_users: page = _).to_string(), i18n!(ctx.1, "Users"), false),
+        (&uri!(instance::admin_email_blocklist:page=_).to_string(),i18n!(ctx.1, "Email blocklist"), true)
+    ])
+    <form method="post" action="@uri!(instance::add_email_blocklist)">
+        @(Input::new("email_address",i18n!(ctx.1,"Email address"))
+        .details(i18n!(ctx.1,"The email address you wish to block. In order to block domains, you can use globbing syntax, for example '*@gmail.com' blocks all addresses from gmail.com"))
```

Comment:
I'm a bit confused as to why there are no spaces after `, `s in function calls, starting here

Reply:
Oh, that's because rustfmt can't touch the templates :)

Fixed.